app/assets/stylesheets/modules/search.scss

```diff
@@ -118,7 +118,7 @@ header .navbar-form .input-group .input-group-btn {
 }
 
 .search-q.form-control.tt-hint {
-  color: white;
+  visibility: hidden;
```

Comment:
This definitely works to make it invisible but from looking at the selectors it's making the input element itself invisible (which I guess is why the white background needed to move to the parent element). I think this will be an accessibility issue in that now screen readers will skip the search input.

I found a new idea that I will put in the ticket.